hieradata/production.yaml

```diff
@@ -621,3 +621,5 @@ shell::shell_prompt_string: 'production'
 
 mongodb::s3backup::backup::s3_bucket: 'govuk-mongodb-backup-s3-production'
 mongodb::s3backup::backup::s3_bucket_daily: 'govuk-mongodb-backup-s3-daily-production'
+
+trade_tariff_uri: 'https://trade-tariff.cloudapps.digital/'
```

Comment:
This isn't the correct URL for Trade Tariff production.  It should be https://tariff-frontend-production.cloudapps.digital/